captcha/fields.py

```diff
@@ -64,19 +64,20 @@ def clean(self, values):
                 recaptcha_response_value == 'PASSED':
             return values[0]
 
-        try:
-            check_captcha = client.submit(
-                recaptcha_challenge_value,
-                recaptcha_response_value, private_key=self.private_key,
-                remoteip=self.get_remote_ip(), use_ssl=self.use_ssl)
-            
-        except socket.error: # Catch timeouts, etc
-            raise ValidationError(
-                self.error_messages['captcha_error']
-            )
+        if self.required == True:
```

Comment:
This can be simplified to avoid the extra indent by doing:

```py
if not self.required:
    return
```

Also you shouldn't do `== True` in Python, just test the variable's truthiness, to allow duck typing.